From 759b92c2ba10f84d50e1509009f31312e2b0fc4d Mon Sep 17 00:00:00 2001
From: Gergely Nagy <algernon@madhouse-project.org>
Date: Sat, 30 Jun 2012 13:09:37 +0200
Subject: [PATCH] Manual page for aalib-config, and aafire(1) updates

Patch by Joey Hess <joeyh@debian.org> and Uwe Hermann <uh1763@hermann-uwe.de>,
24 Oct 2001.

http://bugs.debian.org/cgi-bin/bugreport.cgi?bug=481707
http://bugs.debian.org/cgi-bin/bugreport.cgi?bug=116912
---
 man/aafire.1       |   22 ++++++++++++++++++++--
 man/aalib-config.1 |   52 ++++++++++++++++++++++++++++++++++++++++++++++++++++
 2 files changed, 72 insertions(+), 2 deletions(-)
 create mode 100644 man/aalib-config.1

diff --git a/man/aafire.1 b/man/aafire.1
index 5269fcc..972a491 100644
--- a/man/aafire.1
+++ b/man/aafire.1
@@ -23,10 +23,10 @@ and
 programs.
 .PP
 All of these programs exist to demonstrate the capabilities of the aalib
-library, an ascii art library.
+library, an ASCII art library.
 .PP
 .B aafire
-displays burning ascii art flames.
+displays burning ASCII art flames.
 .PP
 .B aainfo
 displays information about what drivers aalib will use for the display,
@@ -38,6 +38,24 @@ saves a font to a file.
 .PP
 .B aatest
 tests the capabilities of aalib.
+.SH BUGS
+Please report any bugs you find to Jan Hubicka <hubicka@freesoft.cz>.
+.SH LICENSE
+.BR aafire ,
+.BR aainfo ,
+.BR aasavefont ,
+and 
+.BR aatest
+are all covered by the GNU Lesser General Public License (LGPL).
 .SH AUTHOR
+Jan Hubicka <hubicka@freesoft.cz>
+.PP
 This manual page was written by Joey Hess,
 for the Debian GNU/Linux system.
+.SH SEE ALSO
+.BR bb (1),
+.BR aatv (1),
+.BR aview (1),
+.BR asciiview (1),
+.BR aaflip (1),
+.BR xaos (6).
diff --git a/man/aalib-config.1 b/man/aalib-config.1
new file mode 100644
index 0000000..223df54
--- /dev/null
+++ b/man/aalib-config.1
@@ -0,0 +1,52 @@
+.TH AALIB 1
+.SH NAME
+aalib-config - script to get information about the installed version of aalib
+.SH SYNOPSIS
+.B aalib-config
+[\-\-prefix\fI[=DIR]\fP] [\-\-exec\-prefix\fI[=DIR]\fP] [\-\-version] [\-\-libs] [\-\-static-libs] [\-\-cflags]
+.SH DESCRIPTION
+.PP
+\fIaalib-config\fP is a tool that is used to configure to determine
+the compiler and linker flags that should be used to compile
+and link programs that use \fIaalib\fP. It is also used internally
+to the .m4 macros for GNU autoconf that are included with \fIaalib\fP.
+.
+.SH OPTIONS
+.l
+\fIaalib-config\fP accepts the following options:
+.TP 8
+.B  \-\-version
+Print the currently installed version of \fIaalib\fP on the standard output.
+.TP 8
+.B  \-\-libs
+Print the linker flags that are necessary to link a \fIaalib\fP program.
+.TP 8
+.B  \-\-static-libs
+Print the linker flags that are necessary to link a \fIaalib\fP program
+statically.
+.TP 8
+.B  \-\-cflags
+Print the compiler flags that are necessary to compile a \fIaalib\fP program.
+.TP 8
+.B  \-\-prefix=PREFIX
+If specified, use PREFIX instead of the installation prefix that \fIaalib\fP
+was built with when computing the output for the \-\-cflags and
+\-\-libs options. This option is also used for the exec prefix
+if \-\-exec\-prefix was not specified. This option must be specified
+before any \-\-libs or \-\-cflags options.
+.TP 8
+.B  \-\-exec\-prefix=PREFIX
+If specified, use PREFIX instead of the installation exec prefix that
+\fIaalib\fP was built with when computing the output for the \-\-cflags
+and \-\-libs options.  This option must be specified before any
+\-\-libs or \-\-cflags options.
+.SH COPYRIGHT
+Copyright \(co  1998 Owen Taylor
+
+Permission to use, copy, modify, and distribute this software and its
+documentation for any purpose and without fee is hereby granted,
+provided that the above copyright notice appear in all copies and that
+both that copyright notice and this permission notice appear in
+supporting documentation.
+
+Modified for aalib by Joey Hess
-- 
1.7.10.4

